packages/plugins/test/mocha/simulate.spec.ts

```diff
@@ -172,48 +172,11 @@ describe('Simulate', () => {
   })
 })
 
-describe('simulateDeploymentSubtask', () => {
```

Comment:
As long as I understood the purpose of this test correctly, I believe we can drop it since we don't have a `simulateDeploymentSubtask` function anymore. Let me know if you think we do need something to replace it though.